http/oidc/src/main/java/org/wildfly/security/http/oidc/OidcJsonConfiguration.java

```diff
@@ -99,8 +103,9 @@
         ALWAYS_REFRESH_TOKEN,
         REGISTER_NODE_AT_STARTUP, REGISTER_NODE_PERIOD, TOKEN_STORE, ADAPTER_STATE_COOKIE_PATH, PRINCIPAL_ATTRIBUTE,
         PROXY_URL, TURN_OFF_CHANGE_SESSION_ID_ON_LOGIN, TOKEN_MINIMUM_TIME_TO_LIVE,
-        MIN_TIME_BETWEEN_JWKS_REQUESTS, PUBLIC_KEY_CACHE_TTL,
-        IGNORE_OAUTH_QUERY_PARAMETER, VERIFY_TOKEN_AUDIENCE, TOKEN_SIGNATURE_ALGORITHM, SCOPE,
+        MIN_TIME_BETWEEN_JWKS_REQUESTS, POST_LOGOUT_URI, PUBLIC_KEY_CACHE_TTL,
```

Comment:
Just a minor comment, I don't think these are in alphabetical order so we could just add the new param at the end of the list to make it easier to see the changes.

Reply:
done